src/include/non-scalar/hubbard_u.js

```diff
@@ -0,0 +1,7 @@
+import { Property } from "../../property";
+
+export class HubbardUProperty extends Property {
```

Comment:
Maybe think we should define a parent class and inherit here for both U and V: something like  `PropertyPerAtomicOrbitalScalar` and `PropertyPerAtomicOrbitalPairScalar`